frontend/src/metabase/visualizations/echarts/cartesian/model/axis.ts

```diff
@@ -0,0 +1,251 @@
+import d3 from "d3";
+import _ from "underscore";
+import type {
+  DataKey,
+  Extent,
+  SeriesExtents,
+  SeriesModel,
+} from "metabase/visualizations/echarts/cartesian/model/types";
+import type { ComputedVisualizationSettings } from "metabase/visualizations/types";
+import type { SeriesSettings } from "metabase-types/api";
+import { isNotNull } from "metabase/lib/types";
+
+const KEYS_TO_COMPARE = new Set([
+  "number_style",
+  "currency",
+  "currency_style",
+  "number_separators",
+  "decimals",
+  "scale",
+  "prefix",
+  "suffix",
+]);
+
+export function getLineAreaBarComparisonSettings(
```

Comment:
Nit: let's not export any functions in this file except `getYAxisSplit`, since they are all just helper functions for it.